src/snowflake/connector/connection.py

```diff
@@ -1435,9 +1435,10 @@ def is_still_running(status: QueryStatus) -> bool:
         """Checks whether given status is currently running."""
         return status in (
             QueryStatus.RUNNING,
-            QueryStatus.RESUMING_WAREHOUSE,
             QueryStatus.QUEUED,
+            QueryStatus.RESUMING_WAREHOUSE,
```

Comment:
Why did you move this line around?

Reply:
In the enum in constants.py, RUNNING = 0, QUEUED = 5 & RESUMING_WAREHOUSE = 8, so made it align with the order. No big reason other than parity.